features/admin/add_buyer_domain.feature

```diff
@@ -0,0 +1,30 @@
+@add-buyer-domain
+Feature: Admin buyer domains
+
+Scenario: Admin user can log in and add a buyer domain
+  Given I am logged in as the production admin user
+  And I am on the /admin page
+  And I click the 'Add a buyer email domain' link
+  When I enter 'test.com' in the 'new_buyer_domain' field
+  And I click the 'Add' button
+  Then I see a destructive banner message containing 'You cannot add this domain because it already exists.'
+
+Scenario: Admin user can see invalid format message for invalid domain string
+  Given I am logged in as the production admin user
+  And I am on the /admin page
+  And I click the 'Add a buyer email domain' link
+  When I enter 'test' in the 'new_buyer_domain' field
+  And I click the 'Add' button
+  Then I see a destructive banner message containing 'not a valid format'
+
+Scenario Outline: Correct users cannot access the add a buyer domain
```

Comment:
Same again, do we need this or are we happy that unit test coverage is enough?  I'm not sure.